code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma","water vapor","freon","n2o") 
```

Comment:
This list is locked in time, it should not change.
The "hardcoded" part of it's name refers to when gases were... hardcoded, there were these 4 gases only, no more, no less, however Dunc's great atmos refactor (LISTMOS) allowed us to have more gases (which is why we got freon and excited plasma and bz), these new gases are NOT hardcoded, no gases are anymore, this list exists purely to keep old functionality working, if you want a list of all gases, make one, do not modify this one.

(also you're adding "n2o" to it here, but that's already there as "n2")